apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/RDASource.java

```diff
@@ -0,0 +1,25 @@
+package gov.cms.bfd.pipeline.rda.grpc;
+
+import java.time.Duration;
+
+/**
+ * Interface for objects that retrieve objects from some source and pass them to a sink for
+ * processing. All implementations are AutoCloseable since they will generally hold a network or
+ * database connection.
+ *
+ * @param <T> the type of objects processed
+ */
+public interface RDASource<T> extends AutoCloseable {
```

Comment:
Minor naming convention:

```suggestion
/**
 * Interface for objects that retrieve objects from some source and pass them to a sink for
 * processing. All implementations are AutoCloseable since they will generally hold a network or
 * database connection.
 *
 * @param <T> the type of objects processed
 */
public interface RdaSource<T> extends AutoCloseable {
```